C/Coin-OR/Cbc/build_tarballs.jl

```diff
@@ -27,24 +22,32 @@ sed -i s/elf64ppc/elf64lppc/ configure
 mkdir build
 cd build/
 
-export CPPFLAGS="${CPPFLAGS} -DNDEBUG -I${prefix}/include -I$prefix/include/coin"
+export CPPFLAGS="${CPPFLAGS} -DNDEBUG -I${includedir} -I${includedir}/coin"
 if [[ ${target} == *mingw* ]]; then
     export LDFLAGS="-L$prefix/bin"
 elif [[ ${target} == *linux* ]]; then
     export LDFLAGS="-ldl -lrt"
 fi
 
-../configure --prefix=$prefix --build=${MACHTYPE} --host=${target} \
---with-pic --disable-pkg-config --disable-debug \
---enable-shared lt_cv_deplibs_check_method=pass_all \
---with-blas-lib="-lopenblas" --with-lapack-lib="-lopenblas" \
---with-metis-lib="-lmetis" \
---with-coinutils-lib="-lCoinUtils" \
---with-osi-lib="-lOsi -lCoinUtils" \
---with-clp-lib="-lClp -lOsiClp -lCoinUtils" \
---with-cgl-lib="-lCgl -lClp -lOsiClp -lOsi -lCoinUtils" \
---with-coindepend-lib="-lCgl -lClp -lOsiClp -lOsi -lCoinUtils" \
---enable-cbc-parallel
+../configure \
+    --prefix=${prefix} \
+    --build=${MACHTYPE} \
+    --host=${target} \
+    --with-pic \
+    --disable-pkg-config \
+    --disable-debug \
+    --enable-shared \
+    lt_cv_deplibs_check_method=pass_all \
+    --with-asl-lib="-lasl" \
+    --with-blas-lib="-lopenblas" \
+    --with-lapack-lib="-lopenblas" \
+    --with-metis-lib="-lmetis" \
```

Comment:
@tkralphs is this `METIS` needed? Or is it only because of Clp (in which case I can probably remove?)?.

Reply:
I think I answered my question. I tried compiling with `--with-mumps-lib`, and `ldd cbc` doesn't link to MUMPS or METIS, so I guess the flag is just there if you're compiling with Clp simultaneously.